ui/core/src/config.ts

```diff
@@ -60,6 +61,7 @@ export function getConfig(
   const allAssets = [...sifchainAssets, ...ethereumAssets].map(cacheAsset);
 
   const configMap: ConfigMap = {
+    mainnet: parseConfig(mainnnetconfig as ChainConfig, allAssets),
     localnet: parseConfig(localnetconfig as ChainConfig, allAssets),
     monkeybars: parseConfig(monkeybarsconfig as ChainConfig, allAssets),
```

Comment:
You guys might want to change this key to be more generic ie `testnet` as we have dropped monkeybars and are now on merry-go-round, and this is likely to be dropped for a new network very soon.

Reply:
Good call. Done